test/signers/seed.test.js

```diff
@@ -6,6 +6,9 @@ const { remove } = require('fs-extra')
 const path = require('path')
 const store = require('../../compiled/store')
 
+const log = require('electron-log')
+log.transports.console.level = false
```

Comment:
I put this in a bunch of tests as well. wish there was a way to do it globally for tests, maybe in some sort of test config

Reply:
yeah same